main.tf

```diff
@@ -0,0 +1,54 @@
+variable "amis" {
+  type = "map"
+}
+
+# US East (N. Virginia)
+provider "aws" {
+  version = "~> 1.5"
+  shared_credentials_file = "/vagrant/credentials"
```

Comment:
I think this could be a variable at vars.tf :)

Reply:
Done.